packages/documentation/src/components/Header.astro

```diff
@@ -1,21 +1,18 @@
 ---
+import type { Props } from '@astrojs/starlight/props';
```

Comment:
This is the official method of doing Starlight component override, see https://starlight.astro.build/guides/overriding-components/ for documentation.